api/common/helper.go

```diff
@@ -110,3 +110,25 @@ func MakeSubscribeTransaction(wallet vault.Wallet, identifier string, topic stri
 
 	return txn, nil
 }
+
+func MakeGenerateIDTransaction(wallet vault.Wallet, regFee Fixed64, nonce uint64, fee Fixed64) (*Transaction, error) {
```

Comment:
what about renaming `fee` as 'txFee` here and also in NewGenerateIDTransaction to make the difference between these 2 fee more explict?